hvpy/facade.py

```diff
@@ -27,3 +27,106 @@ def getJP2Image(
     """
     params = getJP2ImageInputParameters(date=date, sourceId=sourceId, jpip=jpip, json=json)
     return execute_api_call(input_parameters=params)
+
+
+@add_shared_docstring(getJP2HeaderInputParameters)
+def getJP2Header(
+    id: int,
+    callback: Optional[str] = None,
+) -> Union[bytes, str, Dict[str, Any]]:
+    """
+    Get the XML header embedded in a JPEG2000 image. Includes the FITS header
+    as well as a section of Helioviewer-specific metadata.
+
+    {Insert}
+    Examples
+    --------
+    >>> from hvpy import getJP2Header
+    >>> getJP2Header(id=9838343,callback="xml_header") #doctest: +ELLIPSIS
```

Comment:
```suggestion
    >>> getJP2Header(id=9838343,callback="xml_header")
```
This should be already setup.